### ISSUE TITLE
[Bug]: In MacOS, top bar exhibits a delayed load and makes the window uninteractable.

### ISSUE THREAD

**Issue by hpretila** [th-ch/youtube-music#1709]
### Preflight Checklist

- [X] I use the latest version of YouTube Music (Application).
- [X] I have searched the [issue tracker](https://github.com/th-ch/youtube-music/issues) for a bug report that matches the one I want to file, without success.
- [X] I understand that **th-ch/youtube-music has NO affiliation with Google or YouTube**

### YouTube Music (Application) Version

3.2.2

### Checklists

- [ ] I use the portable version of the YouTube Music Application.
- [ ] I can reproduce this issue in the [official YTM web version](https://music.youtube.com).

### What operating system are you using?

Windows

### Operating System Version

macOS Ventura 13.3.1

### What CPU architecture are you using?

arm64 (including Apple Silicon)

### Reproduction steps

Open the YouTube Client app.

### Expected Behavior

* Expected top bar to be available on application startup.
* To be able to move the application on startup.

### Actual Behavior

1. Initially we have this black screen where you're able to move window by grabbing an invisible top bar.
    
    <img width="1198" alt="image" src="https://github.com/th-ch/youtube-music/assets/3733778/cfa62c60-942e-438f-8a5f-0a241e466e4e">

3. Once YouTube music begins loading, however, you're unable to interact with the window directly.
    * The time when the page starts loading is unclear visually, but you are unable to move the window at this point.
      
      <img width="1198" alt="image" src="https://github.com/th-ch/youtube-music/assets/3733778/cfa62c60-942e-438f-8a5f-0a241e466e4e">
    * Then we see YouTube Music, but no top bar. Hence you're unable to interact with the window.
      
      ![image](https://github.com/th-ch/youtube-music/assets/3733778/793cd1fc-73c0-4308-aa8e-86f9a029cb6b)

4. The top bar does eventually load, that I assume only happens when the rest of the page has loaded.
    
     <img width="1198" alt="image" src="https://github.com/th-ch/youtube-music/assets/3733778/91165815-95db-4183-8577-cbbffcdbb260">

### Enabled plugins

1. Navigation

**Comment by JellyBrick** [comment]
![image](https://github.com/th-ch/youtube-music/assets/16558115/909cc36f-4d2c-4507-a532-d09235c794cf)

Did you enable the `in-app-menu` plugin?

**Comment by hpretila** [comment]
No I haven't. I've enabled it now, and it shortens the time i have to wait for the window to be interactable since it loads in pretty quickly. Still face the same interactability issue though up to this stage, where it hasn't loaded in yet.

![](https://private-user-images.githubusercontent.com/3733778/302009077-793cd1fc-73c0-4308-aa8e-86f9a029cb6b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY5OTQ4NjIsIm5iZiI6MTcwNjk5NDU2MiwicGF0aCI6Ii8zNzMzNzc4LzMwMjAwOTA3Ny03OTNjZDFmYy03M2MwLTQzMDgtYWE4ZS04NmY5YTAyOWNiNmIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDIwMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAyMDNUMjEwOTIyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NTIzZWMyNmZjOGI1Nzc4YTdjNGMxZTJkMzg3Y2I4N2M4YWJiZDIxZWQwZDQ1OTY2M2E1ZWEyZGQ4Mzk2MWUyYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.lBXNrKLdTL-rNVJidfjNVK_ihVL5-okp8A6SFdJNQ9o)

Added observation, it appears that I can resize the window at all of these stages, it's just actually dragging the window that's disabled.
